src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -170,7 +170,8 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
                 $inAttributes = null;
             }
 
-            if (false === $fetchEager = $propertyMetadata->getAttribute('fetchEager')) {
+            // Calls to getAttribute are nested for compatibility with the old camelCased notation
+            if (false === $fetchEager = $propertyMetadata->getAttribute('fetchEager', $propertyMetadata->getAttribute('fetch_eager'))) {
```

Comment:
Should we add a deprecated notice if `$propertyMetadata->getAttribute('fetch_eager')` is not null?

Reply:
It would force every user already using this option to change the naming, IMO it is not worth it.

Reply:
If it's a deprecation, it's not an enforcement 😉

Reply:
In 3.0 it will. But why not.